code/modules/materials/definitions/gasses/material_gas_mundane.dm

```diff
@@ -86,6 +90,7 @@
 		/decl/material/gas/methyl_bromide = 1
 	)
 	value = 0.25
+	uid = "gas_material_methyl_bromide"
```

Comment:
You've mixed up `gas_material` and `material_gas` in these names - might be good to just use `gas` and reduce uid length and potential for inconsistency.